tests/test_api.py

```diff
@@ -217,6 +217,24 @@ async def test_create_work_order_token(
         "email": "john@home.org",
     }
 
+    # mock the access check for the test dataset to revoke access
+
+    httpx_mock.reset(assert_all_responses_were_requested=True)
```

Comment:
Nice, never seen that before, is this properly documented somewhere?

Reply:
Just in code, but because the method does not have an underscore while other methods in the same class have it, I assume it's part of the public API. This is also called automatically in the teardown code of the pytest fixture.

Reply:
It's also referenced/disabled in the tests that use the MockRouter